app.js

```diff
@@ -0,0 +1,24 @@
+$( document ).ready(function(event) {
+
+	$('.readmore').click(function(event){
+		event.preventDefault();
+		$('.readmore').hide();
+		$('#show-this-on-click').slideDown();
+		$('.readless').removeClass('hide');
+	});
+
+	$('.readless').click(function(event){
+		event.preventDefault();
+		$('#show-this-on-click').slideUp();
```

Comment:
You can add a function in slideUp/SlideDown for actions you want to execute when the sliding action has finished

```
$('#show-this-on-click').slideUp(function(){
    $('.readless').hide();
    $('.readmore').show();
});
```

For a better UX